src/os/stat_linux.go

```diff
@@ -1,5 +1,5 @@
-//go:build linux && !baremetal
-// +build linux,!baremetal
+//go:build linux && !baremetal && !unknow_wasm
+// +build linux,!baremetal,!unknow_wasm
```

Comment:
Is the `unknow_wasm` here correct, or should it also be `wasm_freestanding`?

Reply:
yes, a bug here

Reply:
fixed